nistats/tests/test_hemodynamic_models.py

```diff
@@ -215,17 +215,17 @@ def test_design_warnings():
     """ test that warnings are correctly raised upon weird design specification
     """
     condition = ([-25, 20, 36.5], [0, 0, 0], [1, 1, 1])
-    frametimes = np.linspace(0, 69, 70)
+    frame_times = np.linspace(0, 69, 70)
     hrf_model = 'spm'
     with warnings.catch_warnings(record=True):
         warnings.simplefilter("always")
         assert_warns(UserWarning, compute_regressor, condition, hrf_model, 
-                     frametimes)
+                     frame_times)
     condition = ([-25, -25, 36.5], [0, 0, 0], [1, 1, 1])
     with warnings.catch_warnings(record=True):
         warnings.simplefilter("always")
         assert_warns(UserWarning, compute_regressor, condition, hrf_model, 
-                     frametimes)
+                     frame_times)
 
 if __name__ == "__main__":
```

Comment:
just curious, is that here only to be able to do `python this_test.py` rather than `nosetests this_test.py` ? Not sure I see a good reason not to always use the latter.